<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.54.0" />
  <meta name="author" content="Jean Monlong">
  <meta name="description" content="Postdoctoral researcher">

  
  
  
    
  
  
    
    
    <link rel="stylesheet" href="/css/highlight.min.css">
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-113785126-2', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Jean Monlong">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="Jean Monlong">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="/talk/genomeinformatics2014/">

  

  <title>Population-based Detection of Structural Variants in Normal and Aberrant Genomes | Jean Monlong</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">Jean Monlong</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/#about">
            
            <span>Home</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#publications">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#talks">
            
            <span>Talks</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#teaching">
            
            <span>Teaching</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#notes">
            
            <span>Notes</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>

<div class="pub" itemscope itemtype="http://schema.org/CreativeWork">

  
<div class="article-header">
  <img src="/img/headers/churchillcollege.jpg" class="article-banner" itemprop="image">
  
</div>



  <div class="article-container">

    <div class="pub-title">
      <h1 itemprop="name">Population-based Detection of Structural Variants in Normal and Aberrant Genomes</h1>
      <span class="pub-authors" itemprop="author">
        
      </span>
      
      
      
    </div>

    
      <h3>Abstract</h3>
      <p class="pub-abstract" itemprop="text">Structural Variants (SVs) are key mutational elements in evolution and disease. Their detection from high-throughput sequencing (HTS) data has improved substantially but the fraction of false positive predictions remains high. Few efforts have been directed toward problematic regions of the genome such as low-mappability or repeat-enriched regions, which are known to be rich in SVs. We propose to overcome these limitations by using a large set of experiments and a population-based approach for the identification of abnormal regions. Even in low-mappability regions of the genome, abnormal variation from the reference samples will allow the identification of SVs. Most methods for structural variant detection based on HTS data only address the main GC-content and mappability biases but the remaining technical variation might reduce the analysis power. By comparing properly mapped read coverage across multiple samples we demonstrate the presence of a hitherto unknown technical bias in whole-genome sequence data. Because this bias varies between genomic regions and experiments, we show that a general normalization is not sufficient to correct this systematic sample-specific variation and develop a targeted approach. After normalization, we test each region in each sample by computing a Z-test-like score adjusted for multiple-hypothesis testing. We validate our approach on a cancer resequencing project, using normal-tumor concordance across 100 whole-genome sample pairs as well as comparison with SNP-array calls, and a twin study dataset. Compared to other state-of-the-art methods, our population-based approach detects consistently more concordant events with similar specificity. Very few regions of the genome were excluded from the analysis and a number of SVs were detected in regions of low-mappability. Moreover many partial events, likely due to normal contamination in tumor samples or tumor heterogeneity, are exclusively detected by our approach. By integrating over multiple datasets, our approach is liberated from the constraint of assuming a uniform coverage, having to rely on consecutive bins, or requiring complete copy-number changes. We also demonstrate that the same population-based approach can be used to robustly detect abnormal number of read with discordant pair mapping and improve results from orthogonal method, such as Pair-End Mapping or Split-reads.</p>
    

    <div class="row">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading">Date</div>
          <div class="col-xs-12 col-sm-9" itemprop="datePublished">
            Sep 21, 2014
          </div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>

    <div class="row">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading">Event</div>
          <div class="col-xs-12 col-sm-9">
            <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0543-7" target="_blank">
            Genome Informatics
            </a>
          </div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>

    
    <div class="row">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading">Location</div>
          <div class="col-xs-12 col-sm-9">Cambridge, UK</div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>
    

    
    <div class="row" style="padding-top: 10px">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading" style="line-height:34px;">Links</div>
          <div class="col-xs-12 col-sm-9">

            




<a class="btn btn-primary btn-outline" href="/slides/JeanMonlong-GenomeInformatics2014.pdf">
  PDF
</a>







          </div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style">
      <p><blockquote class="twitter-tweet"><p lang="de" dir="ltr">Jean Monlong SV detection in normal and aberrant genomes <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a></p>&mdash; Karyn Meltz (@KMS_Meltzy) <a href="https://twitter.com/KMS_Meltzy/status/514360307461148672?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">JM PopSV: population based approach. Use a set of reference experiments to detect abnormal patterns <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a></p>&mdash; Karyn Meltz (@KMS_Meltzy) <a href="https://twitter.com/KMS_Meltzy/status/514360691105738752?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Monlong: rather than throw out findings in hard-to-map genome tracts, embrace them -- at population scale -- to call elusive SVs. <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a></p>&mdash; Nathan Pearson (@GenomeNathan) <a href="https://twitter.com/GenomeNathan/status/514361161253683201?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Jean Monlong: importance of normalization. Some sample seqs still biased after linear/quantile norm - try PCA-based normalization <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a></p>&mdash; Nancy Yu (@ynancy) <a href="https://twitter.com/ynancy/status/514361710552293376?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">JM: showing a &quot;funky snowman&quot; plot of tumour-normal Z-scores <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a></p>&mdash; Torsten Seemann (@torstenseemann) <a href="https://twitter.com/torstenseemann/status/514361887157682176?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">JM &quot;funky snowman&quot; plot <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a>. Plz add to bingo <a href="https://twitter.com/konradjk?ref_src=twsrc%5Etfw">@konradjk</a></p>&mdash; Karyn Meltz (@KMS_Meltzy) <a href="https://twitter.com/KMS_Meltzy/status/514361912659030016?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">JM PopSV flags regions of SV but need orthogonal method to further characterize <a href="https://twitter.com/hashtag/gi2014?src=hash&amp;ref_src=twsrc%5Etfw">#gi2014</a></p>&mdash; Karyn Meltz (@KMS_Meltzy) <a href="https://twitter.com/KMS_Meltzy/status/514363106441854977?ref_src=twsrc%5Etfw">September 23, 2014</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</p>

    </div>

  </div>
</div>

<div class="container">
  <nav>
  <ul class="pager">
    
    <li class="previous"><a href="/talk/research-day-2014/"><span
      aria-hidden="true">&larr;</span> Population-based detection of Structural Variants in normal and aberrant genomes.</a></li>
    

    
    <li class="next"><a href="/talk/chsgm2015/">Population-based Detection of Structural Variants in Normal and Aberrant Genomes <span
      aria-hidden="true">&rarr;</span></a></li>
    
  </ul>
</nav>

</div>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2020 Jean Monlong &middot; 

      Powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha512-jGsMH83oKe9asCpkOVkBnUrDDTp8wl+adkB2D+//JtlxO4SrLoJdhbOysIFQJloQFD+C4Fl1rMsQZF76JjV0eQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.2/imagesloaded.pkgd.min.js" integrity="sha512-iHzEu7GbSc705hE2skyH6/AlTpOfBmkx7nUqTLGzPYR+C1tRaItbRlJ7hT/D3YQ9SV0fqLKzp4XY9wKulTBGTw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/TweenMax.min.js" integrity="sha512-Z5heTz36xTemt1TbtbfXtTq5lMfYnOkXM2/eWcTTiLU01+Sw4ku1i7vScDc8fWhrP2abz9GQzgKH5NGBLoYlAw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/plugins/ScrollToPlugin.min.js" integrity="sha512-CDeU7pRtkPX6XJtF/gcFWlEwyaX7mcAp5sO3VIu/ylsdR74wEw4wmBpD5yYTrmMAiAboi9thyBUr1vXRPA7t0Q==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
    </script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML" integrity="sha512-tOav5w1OjvsSJzePRtt2uQPFwBoHt1VZcUq8l8nm5284LEKE9FSJBQryzMBzHxY5P0zRdNqEcpLIRVYFNgu1jw==" crossorigin="anonymous"></script>
    
    
    
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
    
  </body>
</html>

